quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/ClusteringPropertyMappers.java

```diff
@@ -34,7 +34,8 @@ public static PropertyMapper[] getClusteringPropertyMappers() {
                 builder().from("cache-config-file")
                         .mapFrom("cache")
                         .to("kc.spi-connections-infinispan-quarkus-config-file")
-                        .description("Defines the file from which cache configuration should be loaded from.")
+                        .description("Defines the file from which cache configuration should be loaded from."
```

Comment:
There's a missing blank. Due to this, the new sentence doesn't start after a blank, but directly after the fullstop (".").

```suggestion
                        .description("Defines the file from which cache configuration should be loaded from. "
```